setup.py

```diff
@@ -69,7 +69,7 @@
         "wikipedia": ["rdflib==4.2.2"],
         # transformers 4.10.0 will break the translation model we used here
         "nlp": ["texar-pytorch>=0.1.4"],
-        "extractor": ["texar-pytorch>=0.1.4"],
+        "extractor": ["texar-pytorch>=0.1.4", "torch>=1.1.0"],
```

Comment:
So:
1. texar-pytorch requires `pytorch`, so we don't have to explicitly say it here
2. to install torch, it is better to ask users to install them by following pytorch instructions, they can choose whether to install the CPU, GPU version, and other things. basically, pytorch installation can be tricky and a single `pip install torch` may not work